adoc/bp_chap_expanded_support.adoc

```diff
@@ -419,6 +419,8 @@ Some actions are not yet supported:
 * Patch and errata support
 * Bare metal installations, PXE booting, and virtual host provisioning
 * Live patching
+* If you use `server.susemanager.fromdir = ...` option in rhn.conf of SUSE Manager, Ubuntu Client Tools will not work.
```

Comment:
I think this feature has a special name in the docs. But I do not remember what it is.

Maybe "Sneaker Network configuration" or "Offline Configuration" or something similar.

Reply:
The reference I found is: https://github.com/SUSE/doc-susemanager/blob/develop/adoc/bp_chap_choosing_dist_scheme.adoc#updating-repositories-on-productname-from-storage-media

What do you think if we rephrase the sentence here as: "If you are using a repository from storage media"?

Reply:
@keichwa @Loquacity what do you think of the above?

Reply:
Wording seems fine to me. I really hope we don't refer to "sneaker networks" in the docs ... :\